### ISSUE TITLE
Support for Door/Entry Sensor

### ISSUE THREAD

**Issue by jayjc303** [fuatakgun/eufy_security#85]
This doesn't seem to support the Eufy Door/Entry sensor. After installing, it doesn't show up on the device list hence I'll assume that it's not supported. All other Eufy devices show up, including Homebase 2, Floodlight Cam and 2K battery doorbell.

It would be great to detect the sensor's open and close state to trigger other automations.

**Comment by fuatakgun** [comment]
created an issue on WS repository: https://github.com/bropat/eufy-security-ws/issues/32

**Comment by fuatakgun** [comment]
this is waiting on update to new version: https://github.com/fuatakgun/eufy_security/issues/59

**Comment by fuatakgun** [comment]
updated both add-on and integration, please give it a try, YOU MUST UPDATE BOTH OF THEM.

**Comment by fuatakgun** [comment]
should be working fine, please update here if it is not working as expected

**Comment by TheRealStark** [comment]
Works great!  Thanks @fuatakgun !

Do you know if the sensor is able to report battery level and if so, have events trigger when battery level runs low?

**Comment by fuatakgun** [comment]
You tell me :-) do you have a battery sensor for your device? Most of the time, i am coding blindly as i don't own all of these devices

**Comment by TheRealStark** [comment]
Yeah, in the Eufy app, it appears it does display the current battery level of the Entry Sensor!

**Comment by fuatakgun** [comment]
I mean, inside home assistant, do you have a battery sensor after using this integration?

**Comment by TheRealStark** [comment]
No, it doesn't.  This is all that appears:

![image](https://user-images.githubusercontent.com/92559968/145685832-afa6fa26-c7cc-435c-a688-379dd7cc649d.png)

**Comment by fuatakgun** [comment]
And screenshot from eufy app please?

**Comment by fuatakgun** [comment]
Hey, please create a separate issue for missing battery sensor. Also dump your attributes of door sensor using developer tools and share screenshot from eufy app.

**Comment by TheRealStark** [comment]
Done: https://github.com/fuatakgun/eufy_security/issues/158
